Games/Niko: Through The Dream/Steam/script.json

```diff
@@ -0,0 +1,7 @@
+{
+	"scriptName": "Steam",
+	"compatibleOperatingSystems": ["MACOSX", "LINUX"],
+	"testingOperatingSystems": ["LINUX"],
```

Comment:
So it should only be set when the script is working already? (Will add a remark to the wiki to clarify).